code/game/objects/structures/crates_lockers/closets/secure/kitchen.dm

```diff
@@ -0,0 +1,158 @@
+		new /obj/item/reagent_container/food/snacks/grown/chili(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/corn(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/eggplant(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/potato(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/tomato(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/whitebeet(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/cherries(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/lime(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/lemon(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/orange(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/banana(src)
+
+//DRy
+
+/obj/structure/closet/secure_closet/fridge/dry
+	name = "dry"
+
+/obj/structure/closet/secure_closet/fridge/dry/Initialize()
+	. = ..()
+	for(var/i = 0, i < 6, i++)
+		new /obj/item/reagent_container/food/snacks/flour(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/condiment/sugar(src)
+
+//grocery
+
+/obj/structure/closet/secure_closet/fridge/groceries
+	name = "Groceries"
+
+/obj/structure/closet/secure_closet/fridge/groceries/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/drinks/milk(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/drinks/soymilk(src)
+	new /obj/item/storage/fancy/egg_box(src)
+	new /obj/item/reagent_container/food/condiment/enzyme(src)
+
+// Kitchen Reserve big quantity stored in boxes
+
+//meat surplus
+/obj/structure/closet/secure_closet/fridge/meat/stock
+	name = "meat"
+
+/obj/structure/closet/secure_closet/fridge/meat/stock/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/storage/box/meat(src)
+
+//fish surplus
+/obj/structure/closet/secure_closet/fridge/fish/stock
+	name = "fish"
+
+/obj/structure/closet/secure_closet/fridge/fish/stock/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/storage/box/fish(src)
+
+//groceries to hold milk in bulk
+/obj/structure/closet/secure_closet/fridge/groceries/stock
+	name = "Groceries"
+
+/obj/structure/closet/secure_closet/fridge/groceries/stock/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/storage/box/milk(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/storage/box/soymilk(src)
+	for(var/i = 0, i < 7, i++)
+		new /obj/item/storage/fancy/egg_box(src)
+	new /obj/item/storage/box/enzyme(src)
+
+//dry storage for dry food only... not a fridge
+/obj/structure/closet/secure_closet/fridge/dry/stock
+	name = "dry"
+
+/obj/structure/closet/secure_closet/fridge/dry/stock/Initialize()
+	. = ..()
+	for(var/i = 0, i < 4, i++)
```

Comment:
Here.

Reply:
change to the correct for(var/i in 1 to 7) type should be fine